modules/viewport-mercator-project/src/web-mercator-utils.js

```diff
@@ -191,15 +201,14 @@ export function getViewMatrix({
 
   // After the rotateX, z values are in pixel units. Convert them to
   // altitude units. 1 altitude unit = the screen height.
-  mat4.scale(vm, vm, [1, 1, 1 / height]);
+  // mat4.scale(vm, vm, [1, 1, 1 / height]);
 
   // Rotate by bearing, and then by pitch (which tilts the view)
   mat4.rotateX(vm, vm, -pitch * DEGREES_TO_RADIANS);
   mat4.rotateZ(vm, vm, bearing * DEGREES_TO_RADIANS);
 
-  if (flipY) {
-    mat4.scale(vm, vm, [1, -1, 1]);
-  }
+  scale /= height;
+  mat4.scale(vm, vm, [scale, scale, scale]);
```

Comment:
Interesting... so view space is in a new pixels/altitude units?

Reply:
So instead of moving the camera closer, I scaled up the view space. Since we don’t manipulate any geometry in the view space, I guess these two approaches are mathematically equivalent?

The reason I did this was because we do it in the orbit viewport, where users supply near/far/focalDistance as fixed values.

Reply:
Haven't reviewed in detail, these are just some general thoughts, ignore if not relevant:
- Some lighting calculations are traditionally done in view space. So I think there are advantages to avoiding being too clever with the view space. Ideally that would be able to  state that it is equivalent to the world space (after translation and rotation), which is what 3D programmers would typically expect. The more precisely we can define the "invariants" of each space (e.g coordinates are in meters like WGS84 etc, the easier we make it for users to reason about these spaces). 
- In the past when refactoring the view matrices I had problems with pixel size being affected. This made point clouds invisible and that is why I added the `focalDistance` prop.

Reply:
The only change to the view space in this PR is to remove the height-dependent z scaling. WebMercator "pixels" do not have a physical meaning.

Our lighting calculation happen in common space which is before the viewMatrix transform. After this PR, common space is no longer zoom dependent.